validator/signature.go

```diff
@@ -45,29 +45,48 @@ func (v *Validator) signDuty(ctx context.Context, decidedValue []byte, role beac
 	case beacon.RoleAttester:
 		s := &proto.InputValue_Attestation{}
 		if err := json.Unmarshal(decidedValue, s); err != nil {
-			return nil, nil, nil, err
+			return nil, nil, nil, errors.Wrap(err, "failed to marshal attestation")
 		}
 		signedAttestation, r, e := v.beacon.SignAttestation(ctx, s.Attestation.Data, duty, shareKey)
+		if e != nil{
+			return nil, nil, nil, errors.Wrap(err, "failed to sign attestation")
+		}
 		retValueStruct.SignedData = s
 		retValueStruct.GetAttestation().Signature = signedAttestation.Signature
 		retValueStruct.GetAttestation().AggregationBits = signedAttestation.AggregationBits
 		err = e
 		sig = signedAttestation.GetSignature()
 		root = r
-	//case beacon.RoleAggregator:
-	//	signedAggregation, e := v.beacon.SignAggregation(ctx, inputValue.GetAggregationData())
-	//	inputValue.SignedData = &proto.InputValue_Aggregation{
-	//		Aggregation: signedAggregation,
-	//	}
-	//	err = e
-	//	sig = signedAggregation.GetSignature()
-	//case beacon.RoleProposer:
-	//	signedProposal, e := v.beacon.SignProposal(ctx, inputValue.GetBeaconBlock())
-	//	inputValue.SignedData = &proto.InputValue_Block{
-	//		Block: signedProposal,
-	//	}
-	//	err = e
-	//	sig = signedProposal.GetSignature()
+	case beacon.RoleAggregator:
```

Comment:
@nivBlox let's remove aggregator and proposal duties for now

Reply:
i sugget to leave it for the next pr to save us to write the same code
@lior-blox